src/MsSqlMicrosoftDataClientAcceptanceTests/When_outbox_in_ado_mode.cs

```diff
@@ -0,0 +1,57 @@
+namespace NServiceBus.AcceptanceTests;
+
+using System.Data.Common;
+using System.Threading.Tasks;
+using System.Transactions;
+using Microsoft.Data.SqlClient;
+using NServiceBus;
+using NServiceBus.AcceptanceTesting;
+using NServiceBus.AcceptanceTests.EndpointTemplates;
+using NUnit.Framework;
+using Persistence.Sql;
+using IsolationLevel = System.Data.IsolationLevel;
+
+[TestFixture]
+public class When_outbox_in_ado_mode : NServiceBusAcceptanceTest
```

Comment:
Is there a better name than "ado mode"?

Reply:
I think that is descriptive enough. For sure better than "native" that we sometimes use.